### PR TITLE
Nav animations

### LINKED ISSUE #57: Animate tile advance
When moving to a new cutout tile, the centerpiece should advance by fading out current galaxies, zooming out, moving to the next tile, and zooming back in while fading in new galaxies.

Currently, the tile simply flicks to the next image.

### PR DESCRIPTION
**Pull request by wgranger**
Describe your changes.
This branch implements several different styling changes:
- Introduce background image
- Introduce corner logos
- Animate movement to new cutout location
- Disable move buttons for 5 seconds with each movement

Most animation code here is done in the code behind rather than XAML due to the heavy lifting of animating navigation movement.

Fixes #57 

# Review Checklist

- [x] Are tests passing?
- [x] Is the build free of output errors?

### CODE REVIEW

File: GalaxyZooTouchTable/GalaxyZooTouchTable/Views/SpaceView.xaml.cs [hunk 1]
```diff
@@ -7,9 +13,117 @@ namespace GalaxyZooTouchTable.Views
...
             InitializeComponent();
+            ViewModel = DataContext as SpaceViewModel;
+            ViewModel.AnimateMovement += AnimateCutoutMovement;
+        }
+
+        void AnimateCutoutMovement(CardinalDirectionEnum direction)
+        {
+            Point transitionPoint;
+
+            switch (direction) {
```
Comment (chelseatroy) [inline GalaxyZooTouchTable/GalaxyZooTouchTable/Views/SpaceView.xaml.cs]: super clear! 🥇

File: GalaxyZooTouchTable/GalaxyZooTouchTable/Views/SpaceView.xaml.cs [hunk 2]
```diff
@@ -7,9 +13,117 @@ namespace GalaxyZooTouchTable.Views
...
+            currentCutoutImageGroup.Children.Add(scaleLarge);
+
+            previousCutoutImageGroup.Children.Add(scaleSmall);
+            previousCutoutImageGroup.Children.Add(scaleLarge);
+
+            CurrentCutoutImage.RenderTransform = currentCutoutImageGroup;
+            PreviousCutoutImage.RenderTransform = previousCutoutImageGroup;
+        }
+
+        void PerformGalaxyFade()
```
Comment (chelseatroy) [inline GalaxyZooTouchTable/GalaxyZooTouchTable/Views/SpaceView.xaml.cs]: We can probably just say "PerformFade" as this is namespaced to the SpaceView, but of course that's extremely minor.

Comment (wgranger) [reply GalaxyZooTouchTable/GalaxyZooTouchTable/Views/SpaceView.xaml.cs]: Makes sense.

**Comment by chelseatroy** [comment]
Functionality check - Works! Yay!

**Pull request merged**